dotCMS/src/main/java/com/dotcms/api/system/event/message/builder/SystemMessage.java

```diff
@@ -67,4 +69,23 @@ public String toString() {
                 ", type=" + type +
                 '}';
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        SystemMessage that = (SystemMessage) o;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'that' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642172)